src/app/ui/editor/moving_slice_state.cpp

```diff
@@ -9,19 +9,24 @@
 #include "config.h"
 #endif
 
-#include "app/ui/editor/moving_slice_state.h"
-
 #include "app/cmd/set_slice_key.h"
+#include "app/cmd/clear_slices.h"
 #include "app/context_access.h"
 #include "app/tx.h"
 #include "app/ui/editor/editor.h"
+#include "app/ui/editor/moving_slice_state.h"
```

Comment:
The '#include' line from the header file corresponding to the current cpp file always appears at the top. This should be reverted.